build_tools/build_detect_platform

```diff
@@ -134,7 +134,7 @@ case "$TARGET_OS" in
         ;;
     FreeBSD)
         PLATFORM=OS_FREEBSD
-        COMMON_FLAGS="$COMMON_FLAGS -fno-builtin-memcmp -D_REENTRANT -DOS_FREEBSD"
+        COMMON_FLAGS="$COMMON_FLAGS -fno-builtin-memcmp -D_REENTRANT -DOS_FREEBSD -D_GLIBCXX_USE_C99 -fpermissive"
```

Comment:
oh, sorry, you are asking about `_GLIBCXX_USE_C99`, it is required, and rocksdb in ports also uses it

https://github.com/freebsd/freebsd-ports/commit/6adb4a5b5e4770e30d9d10fa14e152bf4c8607be